FastSimulation/Configuration/python/Reconstruction_AftMix_cff.py

```diff
@@ -6,209 +6,6 @@
-
-# not commisoned and not relevant in FastSim (?):
-_reco.muonGlobalReco.remove(_reco.muonreco_with_SET)
-
-# not commisoned and not relevant in FastSim (?):
-_reco.muonGlobalReco.remove(_reco.muonSelectionTypeSequence)
-_reco.muons.FillSelectorMaps = False
-
-# FastSim has no template fit on tracker hits
-_reco.globalMuons.GLBTrajBuilderParameters.GlbRefitterParameters.TrackerRecHitBuilder = 'WithoutRefit'
-_reco.globalMuons.GLBTrajBuilderParameters.TrackerRecHitBuilder = 'WithoutRefit'
-_reco.globalMuons.GLBTrajBuilderParameters.TrackTransformer.TrackerRecHitBuilder = 'WithoutRefit'
-_reco.tevMuons.RefitterParameters.TrackerRecHitBuilder = 'WithoutRefit'
-
-# FastSim doesn't use Runge Kute for propagation
-_reco.globalMuons.GLBTrajBuilderParameters.GlbRefitterParameters.Propagator = 'SmartPropagatorAny'
-_reco.globalMuons.GLBTrajBuilderParameters.GlobalMuonTrackMatcher.Propagator = 'SmartPropagator'
-_reco.globalMuons.GLBTrajBuilderParameters.TrackTransformer.Propagator = 'SmartPropagatorAny'
-_reco.GlbMuKFFitter.Propagator = "SmartPropagatorAny"
-_reco.GlobalMuonRefitter.Propagator = "SmartPropagatorAny"
-_reco.KFSmootherForMuonTrackLoader.Propagator = "SmartPropagatorAny"
-_reco.KFSmootherForRefitInsideOut.Propagator = "SmartPropagatorAny"
-_reco.KFFitterForRefitInsideOut.Propagator = "SmartPropagatorAny"
-_reco.tevMuons.RefitterParameters.Propagator = "SmartPropagatorAny"
-
-##########################################
-# FastSim changes to jet/met reconstruction
-##########################################
-# CSCHaloData depends on cosmic muons, not available in fastsim
-_reco.BeamHaloId.remove(_reco.CSCHaloData)
-# GlobalHaloData and BeamHaloSummary depend on CSCHaloData
-_reco.BeamHaloId.remove(_reco.GlobalHaloData)
-_reco.BeamHaloId.remove(_reco.BeamHaloSummary)
-
-############################################
-# deleting modules to avoid clashes with part of reconstruction run before mixing
-############################################
-for _entry in _mod2del:
-    for _key,_value in _reco.__dict__.items():
-        _index = -1
-        if isinstance(_value,cms.Sequence):
-            try:
-                _index = _value.index(_entry)
-            except:
-                pass
-        if _index >= 0:
-            _value.remove(_entry)
-            # removing the last item does not work, and changes the properties of the sequence
-            # so, we detect this changed behaviour and add the sequence to the list of items to be deleted
-            # this is buggy
-            if not isinstance(_value._seq,_SequenceCollection):
-                _mod2del.append(_value)
-
-# and delete
-for _entry in _mod2del:
-    for _key,_value in _reco.__dict__.items():
-        if _entry == _value:
-            delattr(_reco,_key)
-
-############################################
-# the actual import
-############################################
+#All work is now done with the fastSim era
 from Configuration.StandardSequences.Reconstruction_cff import *
```

Comment:
here and in a few more places, is there a reverse equivalent, along the following lines:
 FastSimulation/Configuration will contain only the fastsim modifiers and by itself can not be used.
Instead then the standard include should change to Configuration.StandardSequences.Reconstruction_cff

Reply:
I was considering changing to just the standard includes as a seperate pull request. I wanted to minimize impact on other code (such as cmsDriver) as possible.